weatheril/__init__.py

```diff
@@ -19,8 +19,225 @@
 weather_codes_url = "https://ims.gov.il/{}/weather_codes"
 
 
-class WeatherIL:
+HE_WEATHER_CODES = {
```

Comment:
Refactored this for not creating a new object every time, little bit memory saving